src/Config.php

```diff
@@ -909,6 +909,7 @@ protected function getDefaults()
                 'dbname'         => 'bolt',
                 'prefix'         => 'bolt_',
                 'charset'        => 'utf8',
```

Comment:
@GawainLynch ah, you're right. I forgot this.
ignore my above note ;p

I don't have experience with databases other than mysql, is it ok to have a default "collate" here?